### ISSUE TITLE
More Information

### ISSUE THREAD

**Issue by mahajanakshay017** [RWaltersMA/CosmosDB2MongoDB#1]
@robwma and @RWaltersMA  Can someone give me more information how to use, kafka connector for cosmos to mongo migration. Both are on azure cloud.

**Comment by RWaltersMA** [comment]
The steps in the README walk you through the setup.  If you choose not to use the docker setup as long as you have a kafka connect environment with the connectors installed that should work fine as well

**Comment by mahajanakshay017** [comment]
No, I am using docker enviorment. Can we automate this process with azure build and release pipline. is it possible?
